src/components/files/FilePreview/PreviewLoading.tsx

```diff
@@ -0,0 +1,23 @@
+import { Grid, makeStyles } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import React from 'react';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    minWidth: '200px',
+  },
+}));
+
+export const PreviewLoading = () => {
```

Comment:
Since this rectangle skeleton is never even close to what the result it would be it's probably better just for this case to use a spinner. What do you think?

Reply:
Agreed